adapters/undertone/undertone.go

```diff
@@ -0,0 +1,214 @@
+package undertone
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/prebid/openrtb/v17/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+	"strconv"
+)
+
+type UndertoneAdapter struct {
```

Comment:
We could just call adapter struct. It doesn't have to be exported. 
`type adapter struct {`